### PR TITLE
adds stop validating content for new IA

### PR DESCRIPTION
**Pull request by dawnkelly09**
This PR: adds content for Infrastructure > Onboarding & Offboarding > Stop Validating to align with updated IA.

### CODE REVIEW

**Review by CrackTheCode016 — APPROVED** [review]

File: infrastructure/running-a-validator/onboarding-and-offboarding/stop-validating.md [hunk 1]
```diff
@@ -1,4 +1,43 @@
...
+
+## Purge Validator Session Keys
+
+Purging validator session keys is a critical step in removing the association between your validator account and its session keys, which ensures that your account is fully disassociated from validator activities. The `session.purgeKeys` extrinsic removes the reference to your session keys from the stash or staking proxy account that originally set them.
+
+Here are a couple of important things to know about purging keys:
+
+- **Account Used to Purge Keys** - always use the same account to purge keys you originally used to set them, usually your stash or staking proxy account. Using a different account may leave an unremovable reference to the session keys on the original account, preventing its reaping
+- **Account Reaping Issue** - failing to purge keys will prevent you from reaping (fully deleting) your stash account. If you attempt to transfer tokens without purging, you'll need to rebond, purge the session keys, unbond again, and wait through the unbonding period before any transfer.
```
Comment (eshaben) [inline infrastructure/running-a-validator/onboarding-and-offboarding/stop-validating.md:35]:
```suggestion
- **Account used to purge keys** - always use the same account to purge keys you originally used to set them, usually your stash or staking proxy account. Using a different account may leave an unremovable reference to the session keys on the original account, preventing its reaping
- **Account reaping issue** - failing to purge keys will prevent you from reaping (fully deleting) your stash account. If you attempt to transfer tokens without purging, you'll need to rebond, purge the session keys, unbond again, and wait through the unbonding period before any transfer
```

File: infrastructure/running-a-validator/onboarding-and-offboarding/stop-validating.md [hunk 2]
```diff
@@ -1,4 +1,43 @@
...
+
+## Unbond Your Tokens
+
+After chilling your node and purging session keys, the final step is to unbond your staked tokens. This action removes them from staking and begins the unbonding period (usually 28 days for Polkadot and seven days for Kusama), after which the tokens will be transferable.
+
+To unbond tokens, go to **Network > Staking > Account Actions** on Polkadot.js Apps. Select your stash account, click on the dropdown menu, and choose **Unbond Funds**. Alternatively, you can use the `staking.unbond` extrinsic if you handle this via the staking proxy account.
```
Comment (eshaben) [inline infrastructure/running-a-validator/onboarding-and-offboarding/stop-validating.md:41]:
```suggestion
To unbond tokens, go to **Network > Staking > Account Actions** on Polkadot.js Apps. Select your stash account, click on the dropdown menu, and choose **Unbond Funds**. Alternatively, you can use the `staking.unbond` extrinsic if you handle this via a staking proxy account.
```

**Review by eshaben — COMMENTED** [review]
This guide seems very conversational rather than action-oriented. I think I would prefer to see something more action-oriented, but maybe that's not what is needed here? @polkadot-developers/docs-content-team wdyt?

**Review by eshaben — APPROVED** [review]
This is fine for now. We can revisit later if needed

**Pull request merged**